mirdata/datasets/cipi.py

```diff
@@ -0,0 +1,259 @@
+        Expressiviness path (str): Path of expressiviness features from sound dimension computed with virtuosoNet model.Use torch.load(...) for loading the embeddings.
+        Notes path (str): Path of note features from notation dimension. Use torch.load(...) for loading the embeddings.
+        scores (list[music21.stream.Score]): music21 scores. If the work is splited in several movements the list will contain multiple scores.
+    """
+
+    def __init__(self, track_id, data_home, dataset_name, index, metadata):
+        super().__init__(track_id, data_home, dataset_name, index, metadata)
+        self._data_home = data_home
+
+    @property
+    def title(self) -> str:
+        return (
+            self._track_metadata["work_name"]
+            if "work_name" in self._track_metadata
+            else None
+        )
+
+    @property
+    def book(self) -> str:
+        return self._track_metadata["book"] if "book" in self._track_metadata else None
+
+    @property
+    def URI(self) -> str:
+        return self._track_metadata["URI"] if "URI" in self._track_metadata else None
+
+    @property
+    def composer(self) -> str:
+        return (
+            self._track_metadata["composer"]
+            if "composer" in self._track_metadata
+            else None
+        )
+
+    @property
+    def musicxml_paths(self) -> List[str]:
+        return (
+            list(self._track_metadata["path"].values())
+            if "path" in self._track_metadata
+            else []
+        )
+
+    @property
+    def difficulty_annotation(self) -> str:
+        return (
+            self._track_metadata["henle"] if "henle" in self._track_metadata else None
+        )
+
+    def _check_embedding(self, fpath, file_type: str) -> str:
+        """
+        Verifies the existence of an embedding file and returns its path.
+
+        Args:
+            fpaths (str): The path to the embedding file.
+            file_type (str): The type of the embedding file.
+
+        Returns:
+            str: The path to the embedding file.
+
+        Raises:
+            FileNotFoundError: If the embedding file does not exist.
+        """
+        try:
+            with smart_open.open(fpath):
+                return fpath
+        except FileNotFoundError:
+            raise FileNotFoundError(
+                f"{file_type} embedding {fpath} for track {self.track_id} not found. "
+                "Did you run .download()?"
+            )
+
+    @core.cached_property
+    def fingering(self) -> tuple:
+        path_rh = self.get_path("rh_fingering")
+        path_lh = self.get_path("lh_fingering")
+        return self._check_embedding(path_rh, "Fingering"), self._check_embedding(
+            path_lh, "Fingering"
+        )
+
+    @core.cached_property
+    def expressiviness(self) -> str:
+        path = self.get_path("expressiviness")
```

Comment:
I think this path should be defined in the __ini__ of track, like `self.expressiveness_path = self.bla bla` and you get it here from self. Or is there something that I am missing here?